server/webhook_http_test.go

```diff
@@ -65,6 +65,7 @@ func TestWebhookHTTP(t *testing.T) {
 		ExpectedSlackAttachment bool
 		ExpectedText            string
 		ExpectedFields          []*model.SlackAttachmentField
+		CurrentInstance         bool
```

Comment:
Is it worth just making this a sub-test and running it twice, with the 2 CurrrentInstance settings?

Reply:
I've spent a bit too much time trying to make this work. I looped through currentInstance true and false, but I kept running into problems with the `webhookWrapperFunc` and the `httptest recorder` -- I think they aren't being reset, even if I wrapped the entire `t.Run` function in the loop. 

Can you take a quick look at the current commit? I'm looping at line 185. It seems like it should work, but it doesn't.